dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/MetricsMonitorTest.java

```diff
@@ -98,6 +98,7 @@ public void testEnableDisableFeatures() throws Exception {
     	server.startServer();
     	Log.info(c, testName, server.waitForStringInLog("defaultHttpEndpoint-ssl",60000));
     	Log.info(c, testName, "------- server started -----");
+    	Log.info(c, testName, server.waitForStringInLog("CWWKT0016I: Web application available (default_host): http://.*:.*/metrics/",60000));
```

Comment:
I'd use `assertNotNull("Web application /metrics not loaded", server.waitForStringInLog(regex))` and some characters in your regex are not escaped.  See javadoc for `java.util.regex.Pattern` for details.